packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -780,7 +780,7 @@ const ruleset: any = {
       },
     },
     // RPC Code: RPC-Uri-V1-12
-    trackedExtensionResourcesAreNotAllowed: {
+    TrackedExtensionResourcesAreNotAllowed: {
```

Comment:
Intentional or by mistake?

Reply:
rule name needs to start with a capital alphabet